### ISSUE TITLE
normalizeTerm only replaces first occurrence of special chars

### ISSUE THREAD

**Issue by marcus-at-localhost** [chromawoods/instaFilta2#1]
Hi,
I am using your script and so far it works flawless for my needs. I just stumbled upon the fact that when terms are normalized your function replaces just the first occurrence

`Säbelschnäbler => sabelschnäbler`

Using replace with regular expression and the `/g` flag replaces all ä's in this case.

``` js
    /* Returns a normalized string based on term settings. */
    var normalizeTerm = function(term) {

      /* Check casing setting. */
      if (settings.caseSensitive === false) {
        term = term.toLowerCase();
      }

      /* Check synonym setting. */
      if (settings.useSynonyms) {
        $.each(settings.synonyms, function(i, synonym) {
          $.each(synonym.src, function(i, srcChar) {
           // replace all occurrences of synonym
            var re = new RegExp(srcChar, 'g');
            term = term.replace(re, synonym.dst);
          });
        });
      }
      return term;
    };
```

**Comment by chromawoods** [comment]
Hi, thanks for noticing, I will fix it as soon as I have a moment to spare!

**Comment by chromawoods** [comment]
Fixed as of 2.0.4.
